api/src/main/java/io/opentelemetry/api/trace/attributes/SemanticAttributes.java

```diff
@@ -96,7 +160,7 @@
 
   /**
    * The version string of the VM image as defined in [Version
-   * Attributes](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#version-attributes).
+   * Attributes](README.md#version-attributes).
```

Comment:
this doesn't look right, if line 136 is right.

Reply:
It is wrong, but that is currently a limitation of the md generator: https://github.com/open-telemetry/build-tools/issues/20

Reply:
The links are currently relative regarding the place where the markdown tables are generated in the spec. Nothing we can do about it without fixing the tool first, then we can make the YAML right again.